filament/include/filament/Skybox.h

```diff
@@ -64,6 +64,12 @@ class UTILS_PUBLIC Skybox : public FilamentAPI {
     struct BuilderDetails;
 
 public:
+    enum SkyboxType {
```

Comment:
Filament seems to make use of scoped enums, I think we should follow this convention